doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -386,41 +339,24 @@ Requests to the environment
 
          SearchPattern (forall l : list _, _ l l).
 
-   Patterns need not be linear: you can express that the same expression
-   must occur in two places by using pattern variables `?ident`.
-
 
    .. example::
 
       .. coqtop:: all
 
          SearchPattern (?X1 + _ = _ + ?X1).
 
-   .. cmdv:: SearchPattern @term inside {+ @qualid }
-
-      This restricts the search to constructions defined in the modules
-      named by the given :n:`qualid` sequence.
-
-   .. cmdv:: SearchPattern @term outside {+ @qualid }
+.. cmd:: SearchRewrite @one_term {? {| inside | outside } {+ @qualid } }
 
-      This restricts the search to constructions not defined in the modules
-      named by the given :n:`qualid` sequence.
+   Displays the name and type of all hypotheses of the
+   selected goals (if any) and theorems of the current context whose
+   conclusion is an equality for which one side matches the
+   expression :n:`@one_term`.
 
-   .. cmdv:: @selector: SearchPattern @term
+   For a statement in the form :n:`{* P__n -> } C`, the :n:`P__n` are the *premises*
+   and :n:`C` is the *conclusion*.
```

Comment:
This explanation is useful but kind of odd in this place. It would be useful to be able to refer to words that are well-defined (once in the manual). For instance, in the paragraph above "conclusion" would be clickable and would refer directly to the place containing this explanation.

Reply:
I agree 100%.  This should be moved to another section in another chapter later on.